.github/workflows/build-yocto.yml

```diff
@@ -29,4 +29,6 @@ jobs:
           export SSTATE_DIR=${PERSISTENT_DIR}/sstate-cache
           mkdir -p $DL_DIR
           mkdir -p $SSTATE_DIR
+          kas dump --update --lock --inplace ci/base.yml
```

Comment:
What are we trying to do here. I don't understand why we are making this change. I understand what it does, but not what we want to achieve.

Reply:
We need to make it possible to replicate a task that was successful.
For now we're just going to capture what we need to be able to do this. This can then be used, for example, to create an auxiliary repository that contains exact commit references and not floating ones as is currently the case.

Reply:
I have add jobs artifacts to share the lock file.

Reply:
yeah, we can attach them in GH.